src/query/pipeline/sources/src/processors/sources/input_formats/input_context.rs

```diff
@@ -104,7 +173,7 @@ impl InputContext {
         let format = Self::get_input_format(&file_format_options.format)?;
         let file_infos = Self::get_file_infos(&format, &operator, &plan).await?;
         let splits = format.split_files(file_infos, split_size);
-        let rows_per_block = settings.get_max_block_size()? as usize;
+        let rows_per_block = 1000 * 1000 * 0.8 as usize;
```

Comment:
is `max_block_size` a constant value?

Reply:
`0.8 as usize` is zero, so `rows_per_block` will be zero.

Reply:
thank you very much!